go.mod

```diff
@@ -3,19 +3,17 @@ module sigs.k8s.io/kubebuilder
 go 1.12
 
 require (
```

Comment:
Update the go.mo and go.sum shows not be part of the scope of this PR. 
I'd recommend you revert it and ensure that your branch is rebased with the master.

Reply:
Hi @mariantalla,  

I still not getting why the modules need to be updated with these changes. So, I'd suggest you:

- Ensure that the branch is rebased with master
- Revert the changes of go.mod and go.sum
- Run `make generate`